checkstyle-tester/README.md

```diff
@@ -121,6 +121,20 @@ With all that done, you can now call groovy. **baseBranch** will be your PR bran
 groovy diff.groovy --localGitRepo /home/johndoe/projects/checkstyle --baseBranch i111-my-fix --patchBranch i111-my-fix-mutation --config config.xml --listOfProjects projects-to-test-on.properties
 ```
 
+## Javadoc regression:
```

Comment:
I think this should be placed at the very bottom.
Where it is now isn't a good place anyways. `Attention` that appears afterwards is part `Checkstyle pitest regression` which you accidentally split up by placing it here.

What about naming this section more generic? We could add comments for other areas like `module-info.java` which also cause exceptions that pollute the report sometimes.

Reply:
agree. "Attention" block is for pitest chapter.

>What about naming this section more generic? 

I am not sure. I my mind it is better to have ready to use examples of config. Such README is targeted to new users, who do not want to spend his time to understand all details/nuances. Ones we have few sections like this, we will think about more general chapter.

Reply:
Ok, I am fine holding off on it then.

Reply:
> Where it is now isn't a good place anyways. Attention that appears afterwards is part Checkstyle pitest regression which you accidentally split up by placing your new section here.

@pbludov this isn't done.

Reply:
@rnveach , I moved new section https://github.com/checkstyle/contribution/commit/d6228fc48e3b1f334585f552ecde3140293887cb

Reply:
Thanks.